moped-editor/src/views/projects/projectView/ProjectTeam/ProjectTeamRoleMultiselect.js

```diff
@@ -11,16 +11,38 @@ import {
   Typography,
 } from "@mui/material";
 import { useGridApiContext } from "@mui/x-data-grid-pro";
-
 import theme from "src/theme";
 
-const ProjectTeamRoleMultiselect = ({ id, field, roles, value, error }) => {
+/**
+ * @param {Integer} id - Data Grid row id (same as record id)
+ * @param {String} value - field value
+ * @param {String} field - name of field
+ * @param {[Object]} roles - array of role objects: {project_role_id, project_role_name, project_role_description}
+ * @param {Boolean} hasFocus - does this field have focus
+ * @param {Boolean} error - toggles error style in textfield
+ * @return {JSX.Element}
+ */
+const ProjectTeamRoleMultiselect = ({
```

Comment:
I noticed that this component is logging some errors about duplicate keys. I don't think that this is related to your changes, but this looks like the same thing that happened with the dropdown options in the team advanced search in the list view - the ever growing number of Amenity's issue 📈 

There is a user that I recently added to Moped that has the same name as someone imported during the Access migration. Could you check to see if this seems fixable in this PR? If not, I can create a spin-off.

<img width="722" alt="Screenshot 2025-01-28 at 10 34 51 AM" src="https://github.com/user-attachments/assets/89013e08-2dce-4fe2-a04b-658746759b0f" />

Reply:
thinking about how a Moped user would differentiate who is who when choosing an option - i noticed that these two users are in different workgroups. 

Looking at their emails, it looks like CTM handles this issue by adding integers at the end of emails of people with the same name - mike.dilley, mike.dilley1....